### ISSUE TITLE
add brightness calculation API to SRW

### ISSUE THREAD

**Issue by bruhwiler** [radiasoft/sirepo#201]

**Comment by mrakitin** [comment]
I think I need an example in Python from @ochubar first. Then it can be implemented in Sirepo.

**Comment by mrakitin** [comment]
I requested an example from Oleg.

**Comment by mrakitin** [comment]
From Oleg:
> Brightness is not yet implemented in SRW for Python; it is only available in SRW for Igor Pro. For undulator brightness see menu SRWE->Undulator->Estimate Brilliance->Treating Energy Spread and “Detuning…” (there are similar functions for Bending Magnet and Wiggler sources). Most of  these calculations are implemented in Igor macro language; there are also some tabulated dimensionless “universal functions”. 
First, all these calculation methods have to be ported to SRW under Python (implemented in Py or maybe in C++), and only after this they can be made available in Sirepo.

**Comment by mrakitin** [comment]
To be assigned to Boaz Nash.

**Comment by robnagler** [comment]
We support brightness in Sirepo. When this gets merged into SRW is up to @ochubar.
